core/dir/empty_spec.rb

```diff
@@ -0,0 +1,37 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/common', __FILE__)
+
+describe "Dir.empty?" do
+  ruby_version_is "2.4" do
+    before :all do
+      DirSpecs.create_mock_dirs
+      @empty_dir = DirSpecs.mock_dir "empty_dir"
```

Comment:
You could simply do
```
@empty_dir = tmp("empty_dir")
```
here, the `tmp` helper will make sure the name is unique.